frontends/web/src/utils/event2.ts

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright 2021 Shift Crypto AG
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { apiWebsocket } from '../utils/websocket';
+
+// This type describes the subject of an event.
+type ISubject = string;
+
+// This interface models the events that are received from the backend.
+export interface IEvent {
+    readonly data?: ISubject; // statusChanged registeredChanged attestationCheckDone syncstarted syncdone headersSynced feeTargetsChanged
+    readonly deviceID?: string;
+    readonly meta?: null;
+    readonly type?: string; // device devices account
+    readonly code?: string; // CoinCode
+}
```

Comment:
comments here

`statusChanged registeredChanged attestationCheckDone syncstarted syncdone headersSynced feeTargetsChanged` or `device devices account` are just mental notes of what data is coming in. My plan is to currently keep it raw here and remove, but specify it in `api/subscribe`